Core1Platform/UsingCore1/01b_U2FLogin.md

```diff
@@ -0,0 +1,45 @@
+[!!Set up U2F from the user side](../../MFA/Integration/01_UserSetupActindo.md)
+[!!Set up U2F from the administrator side](../../MFA/Integration/02_AdminSetupActindo.md)
+
+# Log in via U2F
+
+After U2F has been configured for your Actindo account, you will use U2F from your next login to your Actindo account on.
+
+#### Prerequisites
+
+- U2F has been configured for your account, see [Set up U2F from the user side](../UsingCore1/01a_UserSetupActindo.md) and [Set up U2F from the administrator side](../AdministratingCore1/03_AdminSetupActindo.md).
+- You possess a *YubiKey*.
+
+#### Procedure
+
+*Login Actindo*
+
+![Login username](../../Assets/Screenshots/Core1Platform/LoginUserName.png "[Login username]")
+
+1. Enter your username in the *Username* field and click the [LOGIN] button.   
+    The *Password* field is displayed.
+
+    ![Login password](../../Assets/Screenshots/Core1Platform/LoginPassword.png "[Login password]")
+
+2. Enter your password in the *Password* field and click the [LOGIN] button.   
+    The *Click here to start the second factor authentication* button is displayed.
+
+    ![Login U2F](../../Assets/Screenshots/Core1Platform/LoginU2F.png "[Login U2F]")
+
```

Comment:
Again like in 01a - Some keys are different and maybe you'll have to type in your pre-set Pin before touching the U2F Key

Reply:
done